cmd/kyma/alpha/disable/module/module.go

```diff
@@ -157,7 +157,7 @@ func disableModule(modules []v1beta1.Module, name, channel string) ([]v1beta1.Mo
 			continue
 		}
 		if channel != "" {
-			if mod.Channel != channel {
+			if mod.Channel != "" && mod.Channel != channel {
```

Comment:
if the module was found but not in the matching channel, maybe we should warn the user then or log this scenario. WDYT?

Reply:
@jakobmoellersap You mean warn the user that we are disabling the module from the global Kyma channel? because here the global channel is only considered IF the module doesn't have a channel that overwrites it.